### PR TITLE
Suggestion list for inputs in property editor

### PR DESCRIPTION
**Pull request by schoettl**
For issue #143

### CODE REVIEW

File: src/components/OrgFile/components/PropertyListEditorModal/index.js [hunk 1]
```diff
@@ -85,6 +85,12 @@ export default class PropertyListEditorModal extends PureComponent {
       <Drawer onClose={onClose}>
         <h2 className="drawer-modal__title">Edit property list</h2>
 
+        <datalist id="datalist-property-names">
+          <option value="blocks"/>
+          <option value="depends"/>
+          <option value="assignee"/>
```
Comment (schoettl) [inline src/components/OrgFile/components/PropertyListEditorModal/index.js]: @munen could you give me a hint where to get a combined list of all properties of the file from?

In other words, where is the modal called from and where are the arguments passed? Sorry for that beginner question...

Comment (munen) [reply src/components/OrgFile/components/PropertyListEditorModal/index.js]: @schoettl One of the benefits of the used stack (React, Redux) is that you've got great tooling to work on these questions! Both of them have developer tools that you can install to your browser. Using them, you can see that the `Header` component receives this info as props:

image

[Here's](https://github.com/200ok-ch/organice/blob/master/src/components/OrgFile/components/HeaderList/index.js#L127) where that rendering is happening and the props are passed.

If you go further down the rabbit hole, you'll see that the `HeaderList` component get's this information itself through props [here](https://github.com/200ok-ch/organice/blob/master/src/components/OrgFile/components/HeaderList/index.js#L43-L55) and [here](https://github.com/200ok-ch/organice/blob/master/src/components/OrgFile/components/HeaderList/index.js#L143). 

organice implements the redux connect pattern here with the function `mapStateToProps`. This pattern is documented here: https://react-redux.js.org/using-react-redux/connect-mapstate

**Comment by schoettl** [comment]
Missing: Tests and testing

**Comment by schoettl** [comment]
Added unit tests. Tested in Firefox and Chromium on Arch.

Ready to merge?

**Comment by munen** [comment]
@schoettl I did a quick test. For the files I used (for example sample.org), there's property values missing:

image

**Comment by munen** [comment]
Also, auto-complete will always show the one match "Maverick" for this file - even if I type in something completely different. 

Image from iOS

**Comment by schoettl** [comment]
> @schoettl I did a quick test. For the files I used (for example sample.org), there's property values missing:

Thanks for the feedback although I don't quite understand the problem.

1. Only values of properties with the same name are in the suggestion list.
2. No suggestions are displayed if your input already filters them away (although this might depend on the browser)

I think 1. is important, to don't have unrelated suggestions for a property value.

For 2., you can clear the input field and then you shoud see all suggestions for that field.

**Comment by munen** [comment]
> Thanks for the feedback although I don't quite understand the problem.

Very easy. Let me explain: After two rounds of "Scotland Yard" and "Scrabble" I had about 1kg of cheese fondue for dinner. After that, I thought it would be nice to quickly pull in some of your changes. However, after first producing very low blood sugar and then filling my stomach with cheese, I wasn't very smart about new requirements*lol*

I apologize for the false negative!

**Comment by munen** [comment]
After doing another round of tests and a code review, let me rephrase my feedback: Great feature and even greater implementation. This is pretty much a best case PR - clear feature definition, good architecture and all relevant tests already present! I'll happily pull your next Christmas present in^^ 🙏

**Comment by munen** [comment]
I updated the changelog with your new feature and attribution: https://github.com/200ok-ch/organice/commit/ddda0560617369bc6e771ac8178f1dd4d25355b4

:pray: :bowing_man:

**Comment by schoettl** [comment]
Thank you a lot :)
